en/05_Contributing/01_Code.md

```diff
@@ -33,40 +33,50 @@ We ask for this so that the ownership in the license is clear and unambiguous, a
 
 1. Install the project through composer. The process is described in detail in "[Installation through Composer](../getting_started/composer#contributing)".
 
- 		composer create-project --keep-vcs silverstripe/installer ./your-website-folder 4.0.x-dev
+    ```bash
+    composer create-project --keep-vcs silverstripe/installer ./your-website-folder 5.0.x-dev
+    ```
 
 1. Add a new "upstream" remote to the module you want to contribute to (e.g. `cms`). 
 This allows you to track the original repository for changes, and rebase/merge your fork as required.
 Use your Github user name for which you created the fork in Step 1.
 
-		cd framework
-		git remote rename origin upstream
-		git branch --set-upstream-to upstream
-		git remote add -f origin git://github.com/<your-github-user>/silverstripe-framework.git
+    ```bash
+    cd framework
+    git remote rename origin upstream
+    git branch --set-upstream-to upstream
+    git remote add -f origin git://github.com/<your-github-user>/silverstripe-framework.git
+    ```
 
 1. [Branch for new issue and develop on issue branch](code#branch-for-new-issue-and-develop-on-issue-branch)
 
-		# verify current branch 'base' then branch and switch
-		cd framework
-		git status
-		git checkout -b <your-branch-name>
+    ```bash
+    # verify current branch 'base' then branch and switch
+    cd framework
+    git status
+    git checkout -b <your-branch-name>
+    ```
 
 1. As time passes, the upstream repository accumulates new commits. Keep your working copy's branch and issue branch up to date by periodically running a `composer update`.
 As a first step, make sure you have committed all your work, then temporarily switch over to the `master` branch while updating.
 Alternatively, you can use [composer "repositories"](https://getcomposer.org/doc/05-repositories.md#vcs),
 but we've found that dramatically slows down any updates. You may need to [resolve conflicts](https://help.github.com/articles/resolving-merge-conflicts-after-a-git-rebase/).
 
-		(cd framework && git checkout master)
-		composer update
-		(cd framework && git checkout <your-branch-name>) 
-		(cd framework && git rebase upstream/master)
+    ```bash
+    cd framework && git checkout master
+    composer update
+    cd framework && git checkout <your-branch-name>
+    cd framework && git rebase upstream/master
```

Comment:
Do we have `main` branch for framework? Should we alias `master` if not and discontinue master slowly?

```suggestion
    cd framework && git checkout main
    composer update
    cd framework && git checkout <your-branch-name>
    cd framework && git rebase upstream/main
```

Reply:
There is a master branch for most modules but it's not going to be used going forward. Somewhere probably in about a year we'll create new major branches with the appropriate number as the branch name.

Reply:
I'll leave this as is for now - I'll be doing a full pass over this page in a follow-up PR. This current PR is really for easy search-and-replace changes which don't require much thought.